lang/en.lyaml

```diff
@@ -1322,6 +1326,17 @@ en:
         prompt: "Open hubspot.com to install this app?"
         reinstallPrompt: "Open hubspot.com to reinstall this app?"
         decline: "To continue local development of this app, install it in your target test account and re-run {{#bold}}`hs project dev`{{/bold}}"
+      selectHubDBTablePrompt:
+        selectTable: "Select a HubDB table:"
+        enterDest: "Enter the destination path:"
+        errors:
+          noTables: "No HubDB tables found in account {{ accountId }}"
+          allTablesEmpty: "All HubDB tables in account {{ accountId }} are empty"
```

Comment:
I'm able to successfully download empty tables in my current CLI so I think we might not need these empty messages.

Reply:
Addressed in https://github.com/HubSpot/hubspot-cli/pull/1267/commits/78e18349a588f00f0104bb1b53ef5284563207cb